src/DotnetWebApiBench.Api/Startup.cs

```diff
@@ -57,8 +57,8 @@ public virtual void ConfigureServices(IServiceCollection services)
                 .AddFluentValidation(fv =>
                 {
                     fv.RegisterValidatorsFromAssemblyContaining<ProductsController>();
-                });
-            services.AddNorthwindDataAccess(Configuration["ConnectionStrings:Northwind"]);
+                }); 
+            services.AddNorthwindDataAccess(Configuration["ConnectionStrings:Northwind"], Configuration["serverType"]); 
```

Comment:
Lets stick with current configuration parameters format convention. So for example use `db-type` instead of `serverType` to directly indicate that we are referring to a database not to a www server for example.

Reply:
changes applied